src/src/PluginStructs/P098_data_struct.cpp

```diff
@@ -115,7 +117,7 @@ bool P098_data_struct::homePosSet() const
 
 bool P098_data_struct::canRun()
 {
-  if (!homePosSet()) { return false; }
+  if (checkValidPortRange(GPIO_PLUGIN_ID, _config.limitA.gpio) && !homePosSet()) { return false; }
```

Comment:
This changes the behavior, so you can't move to 'home' if the limit switch is defined.

Reply:
Yes, that's right. Isn't that the same behavior as before?
I assume the idea was that findHome() with command "pwmmotorhome" has to be called first. And now this has to be done only if the limitA gpio pin is set.

Reply:
Problem is that in some setups you simply can't move to 'home'.
For example what if the thing you try to move is stuck. 
Then you must be able to move it in the other direction to remove whatever is stuck.

Just a simple example. Someone gets stuck. The first reaction is to jank out the power cord. This does cause the unit to have no homing position anymore.
Now with your code change, you can't free whatever is stuck.

You should always be able to move things. Only move to a position is impossible when the position is not known and as far as I can remember this is how it is programmed....

Reply:
How should we proceed here?

Reply:
I think you're right.
Sorry, I forgot to have another look at this earlier.